charter.md

```diff
@@ -53,15 +53,245 @@ The following responsibilities belong to the Steering Committe, but may be deleg
+be held as soon as possible. [Eligible voters][voter-eligibility] from the most
+recent election will vote in the special election i.e., eligibility will not be
+redetermined at the time of the special election. A committee member elected in
+a special election will serve out the remainder of the term for the person they
+are replacing, regardless of the length of that remainder.
+
+[voter-eligibility]: /elections.md#eligibility-for-voting
+
+### Resignation
+
+If a committee member chooses not to continue in their role, for whatever
+self-elected reason, they must notify the committee in writing.
+
+### Dissolution
+
+If committee members believe that the committee is no longer able to act in
+accordance with the above [mission](#mission), the committee may dissolve by a
+few methods:
+
+* Dissolution by vote
+* Dissolution by loss of committee
+
+See [elections](#elections) for special criteria in the instance of a general
+election triggered by dissolution.
+
+#### Continuity
+
+If the committee is dissolved completely by any mechanism, selected members
+(selection described below) agree to be "continuity advisors" and attend
+meetings of the subsequently elected incoming committee in a non-voting
+capacity for 6 meetings or until the incoming committee dismisses them,
+whichever is first.
+
+##### Selection
+
+At least two (2) and not more than three (3) members shall be selected for
+this duty.
+
+* One shall be the longest-serving outgoing committee member. If multiple, the
+  incoming committee may select between them.
+* The second shall be selected by the incoming committee at their discretion.
+* The incoming committee, at their discretion, may select a third.
+
+#### Dissolution by vote
+
+Dissolution requires an affirmative vote of a **_two-thirds supermajority of
+the [fixed membership of the committee](#composition)_**, and must be unanimous
+when the committee has four (4) or fewer members.
+
+Example:
+
+* 7 (members) / 3 = 2.333...
+* 2.333... * 2 = 4.666...
+* Round up to the nearest whole number (5)
+* An affirmative vote of 5 members would be required to dissolve the committee
+
+The committee must specify a date of dissolution.
+If the committee is dissolved, all seats are vacated on the date specified.
+
+#### Dissolution by loss of committee
+
+If at any time a **_majority of the
+[fixed membership of the committee](#composition)_**'s seats are vacant, the entire committee will be dissolved.
+
+Example:
+
+* 7 (seats) / 2 = 3.5
+* Round up to the nearest whole number (4)
+* If 4 committee seats are vacant, the entire committee will dissolve
+
+The number of vacant seats will be evaluated prior to any replacement of
+previously-seated members.
+
+### Removal
+
+A committee member may be removed from the committee by the other committee
+members. The member should be given the opportunity to resign before they are
+removed.
+
+Removal should be considered for the following reasons:
```

Comment:
Should the "should" be a "must"?

With the current language the steering committee could chose another reason (like they don't like the persons hair do... to provide a fuzz-like input).

Given the "should" along with no requirement to share the reason there could be something the broader community would not approve of that is kept private. In fact, in removal there is nothing that says the reason even needs to be communicated to the removed party.

How much lee-way should the steering committee have over deciding removal of someone who was voted in? What transparency requirements should be here over the reason for that? I ask about transparency so the broad community can be well informed.

Reply:
thanks for your input Matt.

Reply:
@mattfarina you must be kidding me at this point. So i am not going to engage.(like they don't like the persons hair do... to provide a fuzz-like input).

If you use an iota of your skills to protect folks in the community, that would be a better use of your time.

Reply:
> If you use an iota of your skills to protect folks in the community, that would be a better use of your time.

That was probably a bit too harsh. Apologies Matt.